.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,46 @@
+name: golangci-lint
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - main
+  pull_request:
+permissions:
+  contents: read
+  # Optional: allow read access to pull request. Use with `only-new-issues` option.
+  # pull-requests: read
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.17
+      - uses: actions/checkout@v3
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v3
```

Comment:
Can we use here a hashed version of the action? Please see https://github.com/testcontainers/testcontainers-go/pull/556

Reply:
this uses a different image than what you've changed in #556 and I was unable to pin point the specific change within the `golangci/golangci-lint-action`. Are you able to find which hash I'm supposed to use here?

Reply:
resolved via [597cc7e](https://github.com/testcontainers/testcontainers-go/pull/569/commits/597cc7e01a629b40b4613ad339aebc4cd3921415)